### PR TITLE
chore(es6): use ES6 type class for c3DEngine

### PR DESCRIPTION
**Pull request by zarov**
I am doing some things on c3DEngine in another branch and took the opportunity to quickly update the syntax of `c3DEngine`. No changes here apart from this.

### CODE REVIEW

File: src/Renderer/c3DEngine.js [hunk 1]
```diff
@@ -8,258 +8,256 @@ import * as THREE from 'three';
...
-    // The condition below looks weird, but it's correct: querying tabIndex
-    // returns -1 if not set, but we still need to explicitly set it to force
-    // the tabindex focus flag to true (see
-    // https://www.w3.org/TR/html5/editing.html#specially-focusable)
-    if (this.renderer.domElement.tabIndex === -1) {
-        this.renderer.domElement.tabIndex = -1;
-    }
+        if (!this.renderer) {
+            // from THREE/WebGL.js
+            const element = document.createElement('div');
```
Comment (gchoqueux) [inline src/Renderer/c3DEngine.js]: Could you move this html message error in specific file (with message in parameter)?

There's may be a redundancy with console error in line 63.

**Review by gchoqueux — COMMENTED** [review]
Thanks
It's difficult to see if you have only pass `c3Engine` on Class.

File: src/Renderer/c3DEngine.js (hunk 1 above)
Comment (zarov) [reply src/Renderer/c3DEngine.js]: I replaced it by calling the `WEBGL.getErrorMessage()` from `three` directly.

**Comment by zarov** [comment]
Updated. You can use the whitespace flags to hide the indentation changes.

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**